packages/components/src/components/Menu/Menu.tsx

```diff
@@ -71,41 +94,48 @@ const Menu = React.forwardRef<HTMLButtonElement, BoxProps & MenuProps>(
             overflow="hidden"
           >
             {map(items, ({ label, onClick, disabled }, i) => (
-              <MenuItem key={i}>
-                <Box.div
-                  backgroundColor={{
-                    _: "colorBackground",
-                    hover: disabled
-                      ? "colorBackground"
-                      : "colorBackgroundWeakest",
-                  }}
-                >
-                  <Box.button
-                    alignItems="center"
-                    as={FullWidthButton}
-                    disabled={disabled}
-                    justifyContent="flex-start"
-                    onClick={onClick}
-                    padding="d0"
-                    px="d4"
-                    py="d2"
-                    variant="ghost"
-                    w="100%"
+              <MenuItem
+                key={i}
+                render={
```

Comment:
The render was used to avoid the wrong outline added to the menu item:

https://github.com/user-attachments/assets/bf1964f2-bbf5-43ab-a438-29985e35d9d0